presto-docs/src/main/sphinx/connector/mongodb.rst

```diff
@@ -269,3 +269,13 @@ ALTER TABLE
 ^^^^^^^^^^^
 
 The connector supports ``ALTER TABLE RENAME TO`` operation. Other uses of ``ALTER TABLE`` are not supported.
+
+DELETE
+^^^^^^
+
+The MongoDB connector can delete rows from a table. If ``WHERE`` clause is specified, only the matching rows are deleted. Otherwise, all rows from the table are deleted.
+For example, to delete from the table ``lineitem``::
+
+     DELETE FROM lineitem;
```

Comment:
Suggest reversing these two examples to present the two examples in the same order your text describes them in: your text describes "with a WHERE clause" then without, but your examples are in without, then with order.

Reply:
@steveburnett Could you please review now